qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/NoJavadocForOverriddenMethodsCheck/Invalid.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * This is not a real Java class. It won't be compiled ever. It is used
+ * only as a text resource in integration.ChecksIT.
+ */
+
+public final class Invalid {
+
+    @Override
+    public void overridden() {
+    }
+
+    /**
+     * Javadoc.
+     */
+    @Override
+    public void javadoc() {
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void inherited() {
+    }
+
```

Comment:
@mkordas How is a one line Javadoc looks different from the first method Javadoc ?